src/tox_current_env/hooks.py

```diff
@@ -20,11 +20,23 @@ def tox_addoption(parser):
         default=False,
         help="Don't run tests, only print the dependencies",
     )
+    parser.add_argument(
+        "--print-deps-to-file",
+        action="store",
+        dest="print_deps_path",
+        metavar="PATH",
+        default=None,
+        help="Like --print-deps-only, but to a file (overrides the file)",
+    )
 
 
 @tox.hookimpl
 def tox_configure(config):
     """Stores options in the config. Makes all commands external and skips sdist"""
+    if config.option.print_deps_path is not None:
+        config.option.print_deps_only = True
+        with open(config.option.print_deps_path, "w", encoding="utf-8") as f:
+            f.write("")
```

Comment:
Consider [os.truncate](https://docs.python.org/3/library/os.html#os.truncate)(path, 0) instead.

Reply:
actually the previous option also ensure we fail early if the file is not writable and I consider that good. with os.truncate the code gets more complicated:

 - if the file exists, we truncate it
 - if the file does not exist, we need to check if we can create it